.github/workflows/ci.yml

```diff
@@ -9,8 +9,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.3, 2.6, 2.7]
-        graphql_version: ['1.9.19', '1.10.2']
+        ruby: [2.3, 2.6, 2.7, 3.0]
```

Comment:
If we're testing 2.3 and 3.0 now we probably don't need _both_ 2.6 and 2.7 here

Reply:
🤷‍♂️  sure let's drop 2.6